extensions/ql-vscode/src/interface-types.ts

```diff
@@ -127,11 +127,16 @@ export interface NavigatePathMsg {
   direction: number;
 }
 
+export interface UntoggleShowProblemsMsg {
```

Comment:
There aren't many comments in this file currently, but would be nice to at least add one to explain the meaning of each new message, as it relates to user behaviour.